add getting started page (#2)
Automattic/wp-dev-docs: docs/get-started/get-started-with-wordpress-com.md

---
post_title: Get started with Wordpress.com
---

> Wordpress.com offers managed hosting, which helps optimize your website for speed, security, and performance. Managed hosting takes care of many technical aspects of your site for you. WordPress.com has many plans with increasingly powerful features for different site types and budgets.

Reviewer: ```suggestion
Wordpress.com's managed hosting helps optimize your website for speed, security, and performance while taking care of many of the technical aspects of site management. Depending on your needs, you'll find a [range of plans](https://wordpress.com/pricing/) with increasingly powerful features designed to support every site type and budget.
```

> Wordpress.com provides the tools, workflows, and infrastructure you need to build and deploy your website faster. Our platform supports importing existing Wordpress installations, and it has globally distributed, secure infrastucture for serving content from data centers near the user for optimal performance.
> 
> While working on a site, Wordpress.com provides tools for collaboration on your project such as staging sites, coming soon mode, and real-time backups.

Reviewer: ```suggestion
You'll also find the tools, workflows, and infrastructure you need to build and deploy your website faster—from automated imports of existing Wordpress installations to a secure, globally distributed, performance-optimized CDN serving content from the nearest data centers to site vistitors.

Collaboration and iteration on your project are further supported with tools from staging sites and a 'coming soon' mode to real-time backups and one-click restores.
```

## Before you begin

To get started, create an account on Wordpress.com.

> ### [Sign up](https://wordpress.com/start)

Reviewer: I'd recommend not combining links and headings, outside of any internal anchor text. 

Headings are typically used for wayfinding, and it's best practice to link within the body text below. For instance, with the 'Sign up' part of the sentence.

> If you've never used Wordpress.com before, sign up for a new account

Reviewer: They may have used it before _and_ are creating a new account, so it's best to remove assumptions. E.g. 'Sign up for a new account'. I'm not sure you need to break out multiple headings here. It could be as simple as reducing this section to:

## Before you begin
You'll need to [sign up](https://wordpress.com/start) for a new WordPress.com account or [log in](https://wordpress.com/log-in) to an existing account.

> ## Before you begin
> 
> To get started, create an account on Wordpress.com.
> 
> ### [Sign up](https://wordpress.com/start)
> 
> If you've never used Wordpress.com before, sign up for a new account
> 
> ### [Log in](https://wordpress.com/log-in)
> 
> If you already have a account, log in to get started

Reviewer: ```suggestion
## Before you begin

To get started, you'll need to [sign up](https://wordpress.com/start) for a new WordPress.com account or [log in](https://wordpress.com/log-in) to an existing account.
```
